pagopa/io-app
# chore: [IOBP-111,IOBP-116] Addition of LogoPaymentHuge and usage in new Cards components

DEPENDS ON https://github.com/pagopa/io-app/pull/4759
## Short description
|<img width="200" alt="Screenshot 2023-07-13 at 15 36 26" src="https://github.com/pagopa/io-app/assets/60693085/dabea203-eebc-4f54-99dc-e1d2f34bba2b">|<img width="200" alt="Screenshot 2023-07-13 at 15 59 55" src="https://github.com/pagopa/io-app/assets/60693085/7a5a733e-bbd1-479c-807a-1f7a1af67fa7">|

Added new LogoPaymentHuge component and applied it to PaymentCardBig

## List of changes proposed in this pull request
- added LogoPaymentExtended component
- refactored folder structure for the cards section
- added BankLogoOrSkeleton component, which takes an ABI code and displays a loading skeleton until the resource is loaded
- refactored some points of PaymentCardBig
- added new utility function which basically is a template string for the ABI CDN URL
- added the new components to DSLogos

## How to test
run the tests, also navigate to the Design System  (Logos section and Cards section) and make sure the components are rendered as shown in the image and as shown in the design.
